packages/core/docs/advanced/logging.md

```diff
@@ -44,44 +47,56 @@ If not explicitly changed, logging level depends on current environment variable
 
 ## Writing custom loggers
 
-By default we're printing all the events happening in the app in the console  but you can easily write a new logger and use a third-party library (like `consola`) or pass the logs to external service like `Sentry.io`
+By default, we're printing all the events happening in the app in the console but you can easily write a new logger and use a third-party library (like [consola](https://github.com/nuxt-contrib/consola)) or pass the logs to the external service like [Sentry](https://sentry.io/welcome/)
 
```

Comment:
I think winston logger is a way better example here: https://github.com/winstonjs/winston

Reply:
@andrzejewsky you mean instead of Sentry or consola?